tools/webpack/import-all.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Accepts the result of require.context() to add it all to our great big
+ * components object with keys that are the name of components, i.e.
+ *
+ *   // 01-atoms/vue-widget/index.js
+ *   export const name = 'vue-widget';
+ *
+ * results in:
+ *
+ *   components['vue-widget'] = {name: 'vue-widget', enable() {}, disable() {}}
+ *
+ * @param context
+ */
+function importAll(context) {
+  return context.keys().reduce((accumulator, componentPath) => {
+    // console.log(componentPath);
```

Comment:
Stray console log

Reply:
I'm leaving this as a key reference point to debugging.

Reply:
👍